apps/admin-gui/src/app/facilities/pages/resource-detail-page/resource-tags/resource-tags.component.ts

```diff
@@ -48,6 +51,10 @@ export class ResourceTagsComponent implements OnInit {
 
     this.route.parent.params.subscribe(params => {
       this.voId = params['voId'];
+      this.voService.getVoById(this.voId).subscribe(vo => {
+        this.vo = vo;
+        this.updateData();
+      });
```

Comment:
```suggestion
```
completely remove this, after you have object of resource here you have the voId also and there you can construct objeckt of VO this way:
`this.vo  = {
        id: this.resource.voId,
        beanName: "Vo"
      };`

Reply:
Ok, thanks, I've removed vo and void parameter and moved the vo construction to the `setAuthRights`, since it is the only place it is used.